src/azul/chalice.py

```diff
@@ -404,11 +407,22 @@ def _log_request(self):
                 'query': self.current_request.query_params,
                 'headers': self.current_request.headers
             }
-            log.info('Received %s request for %r, with %s.',
+            log.info('Received %s request for %r, with %s',
                      context['httpMethod'],
                      context['path'],
                      json.dumps(request_info, cls=self._LogJSONEncoder))
 
+            body = self.current_request.json_body
+            if not body:
+                msg = ''
+            elif config.debug == 2:
+                body = json.dumps(body, cls=self._LogJSONEncoder)
+                msg = f' ({len(body)} characters)'
+            else:
+                msg = f' (first {str(n := 1024)} characters)'
```

Comment:
```suggestion
                n = 1024
                msg = f' (first {str(n)} characters)'
```
The walrus operator can be useful in an `if` condition, but here seems unnecessary and the code is easier to read without it.